src/website/app/demos/Button/examples/regular.js

```diff
@@ -25,14 +25,19 @@ const DemoLayout = createStyledComponent('div', {
 });
 
 export default {
-  id: 'success',
-  title: 'Success',
-  description: 'Use for positive actions, like when creating something.',
+  id: 'default',
+  title: 'Default',
+  description: `Regular buttons are for triggering different actions around the page.
```

Comment:
buttons or Buttons?

Reply:
or were you suggesting that we just pick `Buttons` or `buttons` for consistency?